Pinta.Core/EventArgs/DocumentCancelEventArgs.cs

```diff
@@ -24,11 +24,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-using System.ComponentModel;
-
 namespace Pinta.Core;
 
-public sealed class DocumentCancelEventArgs : CancelEventArgs
+public sealed class DocumentCancelEventArgs
```

Comment:
This could maybe be renamed to something like DocumentSaveEventArgs since it no longer really does anything cancellation-related?